django/views/debug.py

```diff
@@ -507,6 +516,7 @@ def technical_404_response(request, exception):
         'urlpatterns': tried,
         'reason': str(exception),
         'request': request,
+        'META': get_safe_request_meta(request),
```

Comment:
Some tests seem missing as removing this line entirely doesn't result in any test failures.